### PR TITLE
Ft UI mybrand

### PR DESCRIPTION
**Pull request by Jovz19200**
I am requesting the PR to merge my ft-ui-mybrand branch

### CODE REVIEW

**Comment by teerenzo** [comment]
> I am requesting the PR to merge my ft-ui-mybrand branch

next time explain what you did on your PR

File: index.html [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+                    <img src="images/blog 2.jpg" alt="blog thumbnail 2">
+                    <p>Software Development tools, you must know</p>
+                </a>
+            </div>
+            <div class="blog_1">
+                <a href="blog.html">
+                    <img src="images/blog 3.jpg" alt="blog thumbnail 3">
+                    <p>How much $$ a software developer earn per year in 2024.</p>
+                </a>
+        </div> -->
```
Comment (teerenzo) [inline index.html]: remove unnecessary  comments

File: index.html [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+                        <li> <a href="admin_panel/login.html">Login </a> 
+                            <!-- <i class="fa fa-sort-desc" aria-hidden="true"></i> -->
+    
+                        </li>
+                    </ul>
+                             
+            <!-- </div> -->
+            <!-- <select name="dropdown">
+                <option value="sign_in">Sign in</option>
+            </select> -->
```
Comment (teerenzo) [inline index.html]: this is not necessary

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            <!-- <article> -->
+                <div class="article_head">
+                    <!-- <div class="header">
+                        <h1>Tech Stack for starters 2024</h1>
+                        <p>Choosing the right tech stack as a tech starter depends heavily on 
+                            your specific interests and career goals. There's no single "best" 
+                            tech stack, but some options are more beginner-friendly and versatile
+                            than others. Here are some popular choices for 
+                            tech starters categorized by different paths you might consider</p>
+                    </div> -->
```
Comment (teerenzo) [inline blog.html]: remove this

File: blog.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+                        <li> <a href="admin_panel/login.html">Login </a> 
+                            <!-- <i class="fa fa-sort-desc" aria-hidden="true"></i> -->
+    
+                        </li>
+                    </ul>
+                             
+            <!-- </div> -->
+            <!-- <select name="dropdown">
+                <option value="sign_in">Sign in</option>
+            </select> -->
```
Comment (teerenzo) [inline blog.html]: remove this

File: admin_panel/js/blogs.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    objectData[index] = {id, blog_title, blog_image}
+    document.querySelector('.list_of_blogs').style.display = 'block'
+    document.querySelector('.update_blog').style.display = 'none';
+    readAll()
+
+}
+function del(id){
+    objectData = objectData.filter(rec => rec.id !== id)
+    console.log(id)
+    console.log(objectData)
```
Comment (teerenzo) [inline admin_panel/js/blogs.js]: don't push console logs

**Review by teerenzo — CHANGES_REQUESTED** [review]

**Comment by Jovz19200** [comment]
I have completed adjusting to the requested actions from review

**Pull request merged**